packages/related-keyphrase-suggestions/src/elements/TrendGraph/index.js

```diff
@@ -0,0 +1,90 @@
+import React, { Fragment } from "react";
+import PropTypes from "prop-types";
+
+/* Internal dependencies */
+import TrendGraphTable from "./TrendGraphTable";
+
+/**
+ * Renders a SVG area chart.
+ *
+ * @param {array}    data                      Array of objects with X and Y coordinates for the SVG chart points.
+ * @param {number}   width                     The SVG chart width.
+ * @param {number}   height                    The SVG chart height.
+ * @param {string}   fillColor                 The SVG chart area background color in HEX format.
+ * @param {string}   strokeColor               The SVG chart line color in HEX format.
+ * @param {number}   strokeWidth               The SVG chart line width.
+ *
+ * @returns {JSX.Element} The SVG area chart component.
+ */
+const TrendGraph = ( {
+	data,
+	width = 66,
+	height = 24,
+	fillColor = "#ade3fc",
+	strokeColor = "#498afc",
+	strokeWidth = 1.8,
+} ) => {
+	if ( data.length !== 12 ) {
+		throw new Error( "The data prop must be an array with exactly 12 values." );
+	}
+
+	// When all the y values are zero, make sure the maximumY value is at least 1 to avoid a division by zero later.
+	const maximumYFromData = Math.max( 1, Math.max( ...data.map( point => point ) ) );
+
+	// Reserve some vertical spacing to prevent the SVG stroke from being cut-off.
+	const chartHeight = height - strokeWidth;
+
+	const polylinePoints = data
+		.map( ( point, index ) => {
+			const x = ( index / 11 ) * width;
+			// Add some vertical padding to make sure the line stroke is always drawn within the SVG.
+			const y = chartHeight - ( point / maximumYFromData ) * chartHeight + strokeWidth;
+
+			return `${ x },${ y }`;
+		} )
+		.join( " " );
+
+	// Add points to close the polygon used for the area background.
+	const polygonPoints = `0,${ chartHeight + strokeWidth } ` + polylinePoints + ` ${ width },${ chartHeight + strokeWidth }`;
+
+	return (
+		<Fragment>
+			<svg
+				width={ 66 }
+				height={ 24 }
```

Comment:
These were the props? 🤔

Reply:
Yes, but I don't think we should have them as props, this component is specifically for the keyphrase suggestions

Reply:
right!